queries.sql

```diff
@@ -43,12 +43,17 @@ SET weight_kg = weight_kg * -1
 WHERE weight_kg <0;
 COMMIT;
 
-
-
 SELECT COUNT(*) FROM animals;
 SELECT COUNT(escape_attempts) FROM animals WHERE escape_attempts = 0;
 SELECT AVG(weight_kg) FROM animals;
 SELECT neutered, COUNT(escape_attempts) FROM animals GROUP BY neutered ORDER BY COUNT(escape_attempts) DESC LIMIT 1;
 SELECT species MAX(weight_kg), MIN(weight_kg) FROM animals GROUP BY species;
 SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN DATE '1990-01-01' AND '2000-01-01' GROUP BY species;
 
+SELECT * FROM animals a JOIN owners o ON o.id = a.owner_id WHERE o.full_name = 'Melody Pond';
```

Comment:
- [ ] Please we want to select all animals that belong to Melody Pond it would be better to use LEFT JOIN instead of inner join here.

Reply:
This should be marked as OPTIONAL.

Reply:
Hi @AlucardSanin why do think this should be optional, here we want to choose just the animals that are owned by Melody`What animals belong to Melody Pond?` you are doing an inner join I think it is not the same 😳
![image](https://user-images.githubusercontent.com/74744787/221241736-a3774d58-c85d-49e0-97e8-0a41f586b722.png)

Reply:
Because with simple JOIN (no LEFT JOIN) we got the answer to that

![image](https://user-images.githubusercontent.com/99287916/221244500-bc574e79-d0fc-4530-8508-0762a1a98cb9.png)

This query answer that question, if you wanted it to be displayed differently it's ok, with LEFT JOIN might be better approach, but with just JOIN works, so the use of LEFT JOIN instead of JOIN should be optiona.